src/main/ts/module/README.md

```diff
@@ -2,7 +2,7 @@
 
 A Typescript library for reading, manipulating, and writing OpenAPI and AsyncAPI documents.
 
-Install with `npm install apicurio-data-models`.
+Install with `npm install @apicurio/data-models`.
```

Comment:
I would add script to print message that package is old and please use `apicurio` one

Reply:
`npm deprecate` will do that: https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

Reply:
Are you on that, @craicoverflow or do you need me to do it (deprecate)?

Reply:
I was planning to do that, once this got merged. Thanks